examples/maps/maps.bal

```diff
@@ -1,48 +1,48 @@
 import ballerina/io;
 
 public function main() {
-    // Declare a `map` constrained by type `any`.
+    // Declare a `map` constrained by the type `any`.
     map<any> m;
 
     // You can also declare and initialize a `map` with a map literal.
     map<any> addrMap = { line1: "No. 20", line2: "Palm Grove",
-                         city: "Colombo 03", country: "Sri Lanka" };
+        city: "Colombo 03", country: "Sri Lanka" };
     io:println(addrMap);
 
-    // Retrieve the value for a key using an index operator.
+    // Retrieve a value using index-based access.
     string country = <string>addrMap["country"];
     io:println(country);
 
-    // Retrieve the value for a key from the `map` using the dot (.) operator.
+    // Retrieve a value using field-based access.
     string city = <string>addrMap.city;
     io:println(city);
 
-    // Add or update the value for a key.
+    // Add or update the value of a key.
     addrMap["postalCode"] = "00300";
     addrMap.postalCode = "00301";
     io:println(addrMap);
 
-    // Use the `keys()` function of the `map` to get the keys of the `map` as an array.
+    // Use the `keys()` method to print the keys of the `map` as an array.
     io:println(addrMap.keys());
 
-    // Print the number of keys in the `map`.
+    // Print the number of mappings in the `map`.
     io:println(addrMap.length());
 
-    // Remove a key using the `remove()` function.
+    // Keys can be removed using the `remove()` method.
```

Comment:
`remove()` method will remove both the key and value.

Reply:
```suggestion
    // Mappings can be removed using the `remove()` method.
```